bin.src/generateCoefficients.py

```diff
@@ -6,6 +6,7 @@
 from lsst.sims.movingObjects import Orbits
 from lsst.sims.movingObjects import ChebyFits
 
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Generate Chebyshev polynomial coefficients" +
```

Comment:
Urg, apparently I updated how that worked but neglected to updated the docstring. I'll fix.